files/setup-05-event-processor.sh

```diff
@@ -22,7 +22,52 @@ ESCAPED_VCENTER_USERNAME=$(echo -n ${VCENTER_USERNAME} | python -c 'import sys,j
 ESCAPED_VCENTER_PASSWORD=$(echo -n ${VCENTER_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 ESCAPED_ROOT_PASSWORD=$(echo -n ${ROOT_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 
-if [ "${EVENT_PROCESSOR_TYPE}" == "AWS EventBridge" ]; then
+if [ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]; then
+    if [ -z ${KNATIVE_HOST} ]; then
+      echo "Knative Host was not provided, exiting ..."
+      exit 1
```

Comment:
This is safe and will abort the whole deployment because we have `set -euo pipefail` in `setup.sh` correct?